internal/releasesjson/downloader.go

```diff
@@ -166,6 +166,14 @@ func (d *Downloader) DownloadAndUnpack(ctx context.Context, pv *ProductVersion,
 			return pkgFilePath, err
 		}
 
+		// Determine the appropriate destination file path
+		var dstDir string
+		if isLicenseFile(f.Name) && licenseDir != "" {
+			dstDir = licenseDir
+		} else {
+			dstDir = binDir
+		}
```

Comment:
```suggestion
		dstDir := binDir
		if isLicenseFile(f.Name) && licenseDir != "" {
			dstDir = licenseDir
		}
```

Nitpick - we can simplify this a little bit

Reply:
Good catch! Fixed via 6aa3102